src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Microsoft.VisualStudio.Editor.Razor.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.EditorFeatures.Text" Version="$(Tooling_MicrosoftCodeAnalysisEditorFeaturesTextPackageVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="$(MicrosoftVisualStudioLanguageIntellisensePackageVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language" Version="$(MicrosoftVisualStudioLanguagePackageVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="$(MicrosoftVisualStudioLanguageServerProtocolPackageVersion)" />
```

Comment:
Gave this a lot of thought and I wasn't entirely sure where we were even getting the protocol packages from (what transitive dependency); however, this makes sense thinking about it more. Lets double down here and add all of the [protocol packages](https://github.com/dotnet/razor-tooling/blob/e0bf990067df7c31f29817ff72d33474e05f80e7/src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.csproj#L12-L14) to prevent future whoopsies

Reply:
In case it changes your mind or anyone is interested: the chain is `MS.CodeAnalysis.Razor.Workspaces->MS.CodeAnalysis.ExternalAccess.Razor->MS.CodeAnalysis.LanguageServer.Protocol->MS.VS.LS.Protocol`.

Will add the other packages too.